natlas-server/app/config_loader.py

```diff
@@ -0,0 +1,47 @@
+import config
+
+
+def load_natlas_config(app, db):
+    if not db.engine.has_table("config_item"):
+        return
```

Comment:
If these tables don't exist, can we safely start-up the web server?

Reply:
No, but we also can't safely prevent initialization because we need to allow the app factory to finish. Otherwise `flask db upgrade` will fail to get an app instance and then we can't migrate because we need to migrate, which is a bad place to be. Details are outlined in #402 but I don't think they belong to this change set.